databricks-dbutils-scala/src/main/scala/com/databricks/sdk/scala/dbutils/DBUtils.scala

```diff
@@ -543,19 +543,19 @@ trait DataUtils extends Serializable with WithHelpMethods {
 case class RunId private[dbutils] (id: Long)
 case class CommandContext private[dbutils] (
     /** The run ID of the root run in a workflow. */
-    rootRunId: Option[RunId],
+    rootRunId: Option[RunId] = None,
```

Comment:
We should avoid introducing default values for `CommandContext` fields and initialize explicitly with these values only where necessary.